CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# 9.0.0-next.0 "cardboard-cpu" (2019-10-03)
+
+### cdk
+
+_Highlights_:
+* New infrastructure for creatinv component test harnesses is available in `@angular/cdk/testing`.
+  Documentation for this infrastructure and for specific Angular Material harnesses is in-progress.
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **drag-drop:** dragging styles not reset once dragging is… ([#17150](https://github.com/angular/components/issues/17150)) ([e62e6bd](https://github.com/angular/components/commit/e62e6bd)), closes [#17139](https://github.com/angular/components/issues/17139) |
+| feature |  **testing:** move harness code from cdk-experimental ([#17026](https://github.com/angular/components/issues/17026)) ([bacdc85](https://github.com/angular/components/commit/bacdc85)) |
+
+### material
+
+_Breaking changes:_
+
+* Components can no longer be imported through "@angular/material".
+Use the individual secondary entry-points, such as `@angular/material/button`.
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **a11y:** focus monitor not identifying touch focus inside shadow root ([#17167](https://github.com/angular/components/issues/17167)) ([3673f3d](https://github.com/angular/components/commit/3673f3d)) |
+| bug fix |  **datepicker:** year not formatted in multi-year view button ([#17202](https://github.com/angular/components/issues/17202)) ([ae28fce](https://github.com/angular/components/commit/ae28fce)), closes [#17187](https://github.com/angular/components/issues/17187) |
+| bug fix |  **overlay:** error when trying to add/remove empty string cla… ([#14919](https://github.com/angular/components/issues/14919)) ([5509c23](https://github.com/angular/components/commit/5509c23)) |
```

Comment:
"error when trying to add/remove empty string class"

Reply:
@devversion is it the changelog script that's truncating commit messages?

Reply:
Asked the same question above. looks like it. I'll have a look at this tomorrow.

Reply:
Actually it looks like it just comes from the commit messages which are too long. I'll have a look ig we can somehow compute the full commit title for the changelog.

Reply:
@jelbourn Just check back on this. Looks like it's just an issue with the individual commits. This seems to be because the Github UI truncates the title, and now if you use `squash & merge` it uses the PR title as commit title? (which contains the ellipsis)